src/clinical/donor-repo.ts

```diff
@@ -43,6 +43,11 @@ export const SPECIMEN_SUBMITTER_ID = 'specimen.submitterId';
 export const SPECIMEN_SAMPLE_SUBMITTER_ID = 'specimen.sample.submitterId';
 const AutoIncrement = require('mongoose-sequence')(mongoose);
 
+const DonorModel = mongoose.model<DonorDocument, PaginateModel<DonorDocument>>(
+	'Donor',
+	DonorSchema,
+);
```

Comment:
Moved within file, updated to mongoose-paginate 2
See: https://www.npmjs.com/package/mongoose-paginate-v2#typescript